.travis.yml

```diff
@@ -6,6 +6,7 @@ matrix:
       dist: trusty
       sudo: required
       compiler: gcc
+      env: COVERAGE=1 CMAKE_OPTIONS="-DCMAKE_BUILD_TYPE=Release -DEXIV2_ENABLE_VIDEO=ON -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_BUILD_UNIT_TESTS=ON -DBUILD_WITH_COVERAGE=ON" # All enabled + Coverage
```

Comment:
Does codecov want `COVERAGE=1`?

Reply:
No, it's just an environment variable I added only for the linux-gcc case so that we only generate the coverage report once (see the changes in the file `ci/run.sh`). 

I thought about adding a new element to the matrix just for the coverage, but I think it is fine to just add it for that case.

Reply:
A right, missed that.